components/eventing-controller/pkg/handlers/nats.go

```diff
@@ -90,20 +90,21 @@ func newCloudeventClient(config env.NatsConfig) (cev2.Client, error) {
 
 // SyncSubscription synchronizes the given Kyma subscription to NATS subscription.
 // note: the returned bool should be ignored now. It should act as a marker for changed subscription status.
-func (n *Nats) SyncSubscription(sub *eventingv1alpha1.Subscription, cleaner eventtype.Cleaner, _ ...interface{}) (bool, error) {
+func (n *Nats) SyncSubscription(subscription *eventingv1alpha1.Subscription, cleaner eventtype.Cleaner, _ ...interface{}) (bool, error) {
```

Comment:
i prefer the old `sub` over the new `subscription`. This is in line with https://github.com/golang/go/wiki/CodeReviewComments#variable-names.
I would also prefer if you could do such name changes in a separate PR if they are on a larger scale. In this case one PR that changes all subscription function arguments to subs would be very welcome.